response_operations_ui/controllers/notify_controller.py

```diff
@@ -38,17 +38,16 @@ def _send_message(self, email, template_id, personalisation=None, reference=None
         if reference:
             notification.update({"reference": reference})
 
-        url = f'{self.notify_url}{template_id}'
-
         auth = app.config['SECURITY_USER_NAME'], app.config['SECURITY_USER_PASSWORD']
+        url = f'{self.notify_url}{template_id}'
         response = requests.post(url, auth=auth, json=notification)
-        if response.status_code == 201:
-            logger.info('Notification id sent via Notify-Gateway to GOV.UK Notify.', id=response.json().get("id"))
-        else:
-            ref = reference if reference else 'reference_unknown'
-            raise NotifyError("There was a problem sending a notification to Notify-Gateway to GOV.UK Notify",
-                              reference=ref)
 
+        try:
+            response.raise_for_status()
+        except HTTPError:
+            ref = reference if reference else 'reference_unknown'
+            raise NotifyError('There was a problem sending a notification to Notify-Gateway', reference=ref)
```

Comment:
It would be good to have the response text in the log line (you should just be able to merge this suggestion)

```suggestion
            raise NotifyError('There was a problem sending a notification to Notify-Gateway', message=response.text, reference=ref)
```